src/core/tsi/alts/handshaker/alts_tsi_handshaker.cc

```diff
@@ -80,6 +82,7 @@ typedef struct alts_tsi_handshaker_result {
   grpc_slice serialized_context;
   // Peer's maximum frame size.
   size_t max_frame_size;
+  char* negotiated_transport_protocol;
```

Comment:
nit: Please use `std::string` here.

Reply:
(If you like, please feel free to change the other `char*` uses to `std::string` in a follow-up PR. This file was originally written in C98 and we have not yet migrated it to C++, but rather have been doing incremental cleanup as we go.)